src/store/MoviesContext.jsx

```diff
@@ -8,21 +8,28 @@ export const MoviesContext = createContext();
 export const MoviesContextProvider = (props) => {
     const API_URL=`https://api.themoviedb.org/3/movie/popular?api_key=${API_KEY}`;
     const SEARCH_API_URL=`https://api.themoviedb.org/3/search/movie?api_key=${API_KEY}`;
+    const MOVIE_API_URL=`https://api.themoviedb.org/3/movie`;
     const [movies, setMovies] = useState([]);
```

Comment:
I find that using plurals here, it makes sense for RESTFUL Apis but not for variables and methods, Instead I would call it `[MovieList, setMovieList]`. But its beyond the scope of this PR